### ISSUE TITLE
Module not found issues

### ISSUE THREAD

**Issue by mainrs** [TanklesXL/gladvent#11]
Hello, I am new to Gleam!

I tried to use this package but I cannot get it to work properly. My code is available at https://github.com/mainrs/advent-of-code.

```console
advent-of-code on  main 
❯ gleam run -m gladvent run 1 2 --year=2023  
   Compiled in 0.03s
    Running gladvent.main
error: Failed to run 2023 day 1

cause:
  0: failed to get runner
  1: module aoc_2023/day_1 not found


error: Failed to run 2023 day 2

cause:
  0: failed to get runner
  1: module aoc_2023/day_2 not found
```

I don't really understand what I am doing wrong tbh....

**Comment by TanklesXL** [comment]
hi there! 

so your code actually works properly (i've dropped a copy of the output below), and I was able to run it! 

What I think is happening is due to [a bug with the existing `gleam export package-interface`](https://github.com/gleam-lang/gleam/issues/2898) command that gladvent has used as `v0.7.0` 

You have 2 options here: 

1) downgrade to `gladvent` version `v0.6.2` which has all the same functionality exept does not use the `gleam export package-interface` functionality (the downside being you dont get your functions type-checked by gladvent

2) you run `gleam clean` or delete the gleam build cache in between each time you call `gleam run`

 
the output from running `gleam run -m gladvent run --year=2023 1 2 3`
```
Hint: You can ignore it with an underscore: `_input`.
   Compiled in 5.49s
    Running gladvent.main
Ran 2023 day 1:
  Part 1: Ok(12402)
  Part 2: error: todo - part 2 not implemented in module aoc_2023/day_1 in function pt_2 at line 23

Ran 2023 day 2:
  Part 1: error: todo - part 1 not implemented in module aoc_2023/day_2 in function pt_1 at line 2
  Part 2: error: todo - part 2 not implemented in module aoc_2023/day_2 in function pt_2 at line 6

Ran 2023 day 3:
  Part 1: error: todo - part 1 not implemented in module aoc_2023/day_3 in function pt_1 at line 2
  Part 2: error: todo - part 2 not implemented in module aoc_2023/day_3 in function pt_2 at line 6
```

**Comment by TanklesXL** [comment]
@mainrs I've actually just published `gladvent v0.7.2` which should fix this, please run `gleam update` to fetch the latest version and give it a shot :D
